### DISCUSSION TITLE
Update from RavenCore 1.1 to Haa fails.

### DISCUSSION THREAD

**Discussion by lexxnsk** [RavenSystem/esp-homekit-devices#2164] [Q&A] [answered]
Is it possible to update from old ravencore 1.1 firmware to the latest haa via OTA?
I use Eve app, click firmware update and reboot, device reboots but firmware version remains the same (1.1)

**Comment by RavenSystem** [comment, marked as answer]
It is not possible because GitHub servers did some changes. You must flash it with a wire connection.

**Reply by lexxnsk** [reply]
Damn, this is sad... That sonoff is hidden above a dropped ceiling
